.github/workflows/ci.yml

```diff
@@ -73,7 +72,7 @@ jobs:
         shell: bash
         run: |
           DTS_VER="${{steps.dts-ver.outputs.DTS_VER}}"
-          ssh -i ~/.ssh/dts-ci-key builder@10.1.40.2 "mkdir -p boot/dts/${DTS_VER}"
+          ssh -i ~/.ssh/dts-ci-key -F ~/.ssh/config_deploy builder@10.1.40.2 "mkdir -p boot/dts/${DTS_VER}"
```

Comment:
Is this config needed in ssh and scp? And I'm asking about every line where you changed it.
Currently when you are creating `config_deploy` it only has config for `git.3mdeb.com`. If it's needed then this config won't work, if it isn't then we don't need this part.

Reply:
You're right 3c838057978e2039e19ae27cd9d7eb9cf189ab8e